### PR TITLE
Cyclops Mining Vessel - Second Take

### PR DESCRIPTION
**Pull request by Alberyk**
This pr revives #16071 while fixing its issues.

### CODE REVIEW

**Review by NonQueueingMatt — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

File: maps/away/ships/heph/cyclops/cyclops_ghostroles.dm [hunk 1]
```diff
@@ -0,0 +1,106 @@
+//Cyclops Crew
+/datum/ghostspawner/human/cyclops_crew
+	short_name = "cyclops_crew"
+	name = "Hephaestus Cyclops Mining Crew"
+	desc = "Crew the Hephaestus Mining Vessel."
+	tags = list("External")
```
Comment (DreamySkrell) [inline maps/away/ships/heph/cyclops/cyclops_ghostroles.dm:6]: `welcome_message` should be set on all new ghostspawners as discussed here <https://github.com/Aurorastation/Aurora.3/pull/16248#issuecomment-1521304132>

File: maps/away/ships/heph/cyclops/cyclops_mining_ship.dm [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+	icon_state = "tramp"
+	moving_state = "tramp_moving"
+	colors = list("#BAB86C", "#8B4000")
+	max_speed = 1/(2 SECONDS)
+	burn_delay = 1 SECONDS
+	vessel_mass = 5000
+	fore_dir = SOUTH
+	vessel_size = SHIP_SIZE_SMALL
+	initial_restricted_waypoints = list(
+        "Cyclops Shuttle" = list("nav_hangar_cyclops")
```
Comment (DreamySkrell) [inline maps/away/ships/heph/cyclops/cyclops_mining_ship.dm:30]: Please add RP fluff fields like `designer`, `weapons`, `shiptype`, as described in `code\modules\overmap\overmap_object.dm`

File: maps/away/ships/heph/cyclops/cyclops_ghostroles.dm [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+/datum/ghostspawner/human/cyclops_crew/captain
+	short_name = "cyclops_captain"
+	name = "Hephaestus Cyclops Captain"
+	desc = "Act as the Hephaestus Mining Vessels Captain"
+
+	spawnpoints = list("cyclops_captain")
+	max_count = 1
+
+	outfit = /datum/outfit/admin/cyclops_crew/captain
+	possible_species = list(SPECIES_HUMAN, SPECIES_HUMAN_OFFWORLD, SPECIES_TAJARA, SPECIES_TAJARA_MSAI, SPECIES_TAJARA_ZHAN, SPECIES_SKRELL, SPECIES_SKRELL_AXIORI, SPECIES_UNATHI)
```
[the remark this answers is not in the document]
Comment (RustingWithYou) [reply maps/away/ships/heph/cyclops/cyclops_ghostroles.dm:71]: I can't speak for Tajara, but given the close ties between Hephaestus and Unathi I think that Unathi being allowed as captains is fine from a lore POV.

**Review by DreamySkrell — CHANGES_REQUESTED** [review]
Code and mapping looks fine now I think. If changes below are addressed, just ping me and I'll approve.

Though the ship should have a second external airlock on the west side, as right now it has just one on the east side. Multiple airlocks make EVA less of a pain, and at least two are needed so if some shuttle is docked to one, you can still use the other. I'd put the second external airlock near the ore processors - it can be one of the automatic external airlock spawners.

This is optional, but: I'd also add some catwalks/lattices on the exterior, so the ship looks a bit more industrial, and to make moving around in EVA a bit easier.

**Review by DreamySkrell — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**